### ISSUE TITLE
height settings should be z positions, hovering should be relative

### ISSUE THREAD

**Issue by jerkey** [hzeller/rpt2pnp#1]
tl;dr Z_HOVERING should be relative to the height of the board or tape, whichever is higher, in case you have a 10mm thick piece of wood or something that the board and tape are on, hovering should be Xmm above the other stuff, rather than Xmm absolute Z position.

presently,  #define Z_HOVERING 10 is an absolute hovering position.  but in actuality, assuming people use the homing on their machine (which is ideal) rpt2pnp should be talking about z positions rather than "thickness of board" and thickness of tape.

i envision a future where a printed plastic deck is made for making boards, perhaps something that can be snapped into alignment holes on the printing platform, that has places for tapes and the board or boards.  This will ensure relative positioning for the board from the tapes.
